ci_scripts/check_api_label_cn.py

```diff
@@ -0,0 +1,86 @@
+import sys
+import os
+import re
+
+# check file's api_label
+def check_api_label(rootdir, file):
+    real_file = rootdir + file
+    with open(real_file, 'r', encoding='utf-8') as f:
+        first_line = f.readline()
+    if first_line == en_label_creater(file):
+        return True
+    return False
+
+
+# path -> api_label(the fist line 's style)
+def en_label_creater(file):
+    result = re.sub("api/", "", file)
+    result = re.sub("_cn.rst", "", result)
+    result = re.sub('/', "_", result)
+    result = '.. _cn_' + result + ':'
```

Comment:
```suggestion
    result = f'.. _cn_{result}:'
```

字符串操作使用 f-string 可读性会非常好～